hypervisor/boot/uefi/uefi_boot.c

```diff
@@ -25,7 +25,8 @@ static void efi_spurious_handler(int32_t vector)
 static int32_t uefi_sw_loader(struct acrn_vm *vm)
 {
 	int32_t ret = 0;
-	struct acrn_vcpu *vcpu = get_primary_vcpu(vm);
+	/* get primary vcpu */
+	struct acrn_vcpu *vcpu = &(vm->hw.vcpu_array[BOOT_CPU_ID]);
```

Comment:
use vcpu_from_vid[BOOT_CPU_ID] ?  Same for others

Reply:
Hi Eddie, I fixed it in latest version. Could you please help review it again? Thanks!